src/main/java/com/exasol/repository/JavaMavenGitRepositoryContent.java

```diff
@@ -27,11 +31,10 @@ protected JavaMavenGitRepositoryContent(final GHRepository repository, final Str
 
     @Override
     public synchronized String getVersion() {
```

Comment:
Again lazy loading is not necessary here. Try to avoid it to improve the software's safety.

When is the right time to use lazy loading?
1. If the process of loading the data is expensive (takes long, uses a lot of resources) an
1. The data or object being loaded is only needed in exceptional cases
Imagine having a web UI that displays that 20 tickets are associated with a pull request. Normally that information is enough for users. If you hit the "details" button, the UI then lazily loads and caches the contents of all 20 tickets.

Reply:
I just didn't want to get everything in the constructor

Reply:
And now a constructor throws an exception if something wrong with parsing. I'm not sure that it's a good practice. Especially if I add more fields and also initialize them in the constructor.